pom.xml

```diff
@@ -15,7 +15,7 @@
 		<revision>1.0.0-SNAPSHOT</revision>
 
 		<!-- DEPENDENCIES VERSION -->
-		<jdk.version>1.8</jdk.version>
+		<jdk.version>17</jdk.version>
```

Comment:
Please also adapt the Maven Compiler Plugin to use release property instead of source/target property